src/file-handlers.ts

```diff
@@ -170,13 +170,28 @@ function includeFile(path: string, options: EtaConfig): [TemplateFunction, EtaCo
  * ```
  */
 
+function renderFile(
+  filename: string,
+  data: DataObj,
+  config?: PartialConfig,
+): Promise<string>
+
+function renderFile(
+  filename: string,
+  data: DataObj,
+  config: PartialConfig,
+  cb: CallbackFn
+): void
+
 function renderFile(
   filename: string,
   data: DataObj,
   config?: PartialConfig,
   cb?: CallbackFn
 ): Promise<string> | void
 
+function renderFile(filename: string, data: DataObj, cb: CallbackFn): void
+
 function renderFile(filename: string, data: DataObj, cb?: CallbackFn): Promise<string> | void
```

Comment:
Could we remove this line?

Reply:
yes, should be resolved, see https://github.com/eta-dev/eta/pull/199/commits/fcc1845dc1da2d105283e50aa44f60350ec010d6